learning-team/2023/2023-09-25.md

```diff
@@ -0,0 +1,54 @@
+# September 25th, 2023
+
+
+Facilitator & Note Taker:
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+
+- [ ] Chris Manson (CM)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+- [ ] Chris Ng (CN)
+- [ ] Anne-Greeth Schot-van Herwijnen (AG)
+
+### Guests
+
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+[Open pull requests](https://help-wanted.emberjs.com/pull-requests)
+
+[RFC review app](https://rfcs.emberjs.com/)
+
+[RFC PR Queue](https://github.com/emberjs/rfcs/pulls)
+
+## Topics
+
+### Release Week
+
+### Polaris info page followups?
+
+### EmberData | deprecate legacy finder support RFC and how to implement in Guides (JG 15 min)
+
+- EmberData is introducing an RFC to deprecate legacy finder support: [RFC 964](https://github.com/emberjs/rfcs/pull/964)
+- There is a desire to get this new approach documented in the Guides or somewhere sooner rather than later
+- The RFC itself includes examples that could serve as a Guides/Cheatsheet examples
+
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+
+## Ember Times suggestions
+
+Add writeup ideas / call for contributors / suggestions to [the latest PR](https://github.com/ember-learn/ember-blog/pulls?q=is%3Aopen+is%3Apr+label%3A%22%F0%9F%97%9E+embertimes%22%20or%20#support-ember-times) or [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on Discord.
```

Comment:
fyi there is https://github.com/ember-learn/ember-blog/pull/1305 for ember-times but I can't add `support-ember-times` label so it's not listed in this query

Reply:
Thanks @SergeAstapov, I just merged that in!